src/go/k8s/tests/e2e/update-image-tls-client-auth-external-ca/01-assert.yaml

```diff
@@ -1,12 +1,109 @@
-apiVersion: batch/v1
-kind: Job
+apiVersion: apps/v1
+kind: StatefulSet
 metadata:
-  name: create-schema-update-img-external-client-ca
+  name: update-img-external-client-ca
 status:
+  readyReplicas: 1
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: update-img-external-client-ca-proxy-selfsigned-issuer
+status:
+  conditions:
+    - reason: IsReady
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: update-img-external-client-ca-proxy-root-issuer
+status:
+  conditions:
+    - reason: KeyPairVerified
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: update-img-external-client-ca-proxy-root-certificate
+status:
+  conditions:
+    - reason: Ready
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: update-img-external-client-ca-proxy-api-node
+status:
+  conditions:
+    - reason: Ready
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: update-img-external-client-ca-schema-registry-selfsigned-issuer
+status:
+  conditions:
+    - reason: IsReady
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: update-img-external-client-ca-schema-registry-root-issuer
+status:
+  conditions:
+    - reason: KeyPairVerified
+      status: "True"
+      type: Ready
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: update-img-external-client-ca-schema-registry-root-certificate
+status:
+  conditions:
+    - reason: Ready
+      status: "True"
+      type: Ready
```

Comment:
Shouldn't this schema registry related assertion be moved to `00-assert.yaml`?

Reply:
My understanding is they are initiated by the operator. The cert is self signed.; they are always generated. Although the cert in step 0 is also self signed, but it simulates the scenario that the certs are issued by e.g. LetsEncrypt, and provided before the cluster is created.

Reply:
Ok